graylog2-server/src/main/java/org/graylog2/alerts/AbstractAlertCondition.java

```diff
@@ -51,15 +51,15 @@ public String toString() {
 
     protected final String id;
     protected final Stream stream;
-    protected final Type type;
+    protected final String type;
```

Comment:
How about getting rid of the `Type` enum completely and using a static `String` field on each alert condition?